src/TheThingsNetwork.cpp

```diff
@@ -16,6 +16,9 @@
 #define MAC_TX_OK "mac_tx_ok"
 #define MAC_RX "mac_rx"
 
+#define CHECK_CONFIGURATION "Check your coverage, keys and backend status."
+#define ERROR_KEY_LENGTH "One or more keys are of invalid length."
```

Comment:
yes it should maybe merge #144 before, this PR and add those error message to the list.